contracts/voting-verifier/src/query.rs

```diff
@@ -48,6 +48,26 @@ pub fn message_status(
     ))
 }
 
+pub fn poll_response(deps: Deps, poll_id: PollId) -> Result<PollResponse, ContractError> {
+    let poll = POLLS.load(deps.storage, poll_id)?;
+    let data = match poll {
+        Poll::Messages(_) => {
+            PollData::Messages(poll_messages().idx.load_messages(deps.storage, poll_id)?)
```

Comment:
Can we add a check here that the number of messages matches the number of tallies in the `poll` object? An assert is fine.